src/librbd/cache/pwl/ImageCacheState.h

```diff
@@ -7,8 +7,8 @@
 #include "librbd/ImageCtx.h"
 #include "librbd/cache/Types.h"
 #include <string>
+#include "common/ceph_json.h"
```

Comment:
Because I modified the constructor and introduced `json_spirit::mObject`, I had to add ceph_json.h here. Is there a better way? like `class JSONFormattable`.

Reply:
@tchaikov do you know it?

Reply:
or I modify the constructor to remove the parameter(`json_spirit::mObject`), then call a new decode func in constructor.